modules/imaging_qc/php/imaging_qc.class.inc

```diff
@@ -267,6 +267,14 @@ class Imaging_QC extends \NDB_Menu_Filter
         $scans_done = [];
         $acq_IDs    = [];
         foreach ($scan_types as $scan_type) {
+            // Only include scan types that appear in the MRI Parameter Form
+            $scanTypeInMRIPF = $db->columnExists('mri_parameter_form', $scan_type . "_Scan_done");
+
+            if (!$scanTypeInMRIPF) {
```

Comment:
Is this something that should generate a warning?

Reply:
not sure that it should generate a warning since you will get warning for every processed or derived scan types. This if statement is to ensure that the query is restricted to scans that are supposed to be acquired according to the mri_parameter_form.